src/os_esp32.cc

```diff
@@ -740,21 +740,17 @@ void OS::heap_summary_report(int max_pages, const char* marker) { }
 
 #endif // def TOIT_CMPCTMALLOC
 
-static const int TOIT_IMAGE_DATA_SIZE = 1024;
-static const int TOIT_CONFIG_IMAGE_SIZE = TOIT_IMAGE_DATA_SIZE - UUID_SIZE - sizeof(uint32);
-
 class ImageData {
  public:
   uint32 image_pad = 0;
-  uint32 image_magic1 = 0x7017da7a;  // "Toitdata"
-  // The data between image_magic1 and image_magic2 must be a multiple of 512
+  uint32 image_magic1 = 0x7017da7a;  // "toitdata"
+  // The data between image_magic1 and image_magic2 must be less than 256
   // bytes, otherwise the patching utility will not detect it. Search for
   // 0x7017da7a. Note when updating this restriction is baked into the SDK that
   // you are updating *from* so it can't be fixed without multiple SDK updates.
```

Comment:
Is the second part of the comment still true?
Aren't we patching from the SDK that is currently built?